fortran/plumed_f08.f90.template

```diff
@@ -237,32 +226,23 @@ module plumed_f08_module
 
      ! we then define all the functions needed for overloading
 
-     subroutine plumed_f_cmd_ptr(p,key,val,dummy,error,const,nocopy)
```

Comment:
Also here, there's not reason to keep the dummy argument in this internal call, so good idea removing it